.scalafmt.conf

```diff
@@ -0,0 +1,28 @@
+version = 3.1.1
+runner.dialect=scala212
+project.git=true
+
+align.preset = none
+align.openParenDefnSite = false
+align.openParenCallSite = false
+align.stripMargin = true
+align.tokens = []
+assumeStandardLibraryStripMargin = true
+danglingParentheses.preset = false
+docstrings.style = Asterisk
+docstrings.wrap = no
+importSelectors = singleLine
+indent.extendSite = 2
+literals.hexDigits = Upper
+maxColumn = 100
+newlines.source = keep
+newlines.topLevelStatementBlankLines = []
+optIn.configStyleArguments = false
+rewrite.imports.groups = [
+  ["javax?\\..*"],
+  ["scala\\..*"],
+  ["(?!com\\.aliyun\\.emr\\.rss\\.).*"],
+  ["com\\.aliyun\\.emr\\.rss\\..*"]
+]
```

Comment:
Seems can't keep `com.aliyun.emr.rss` in the end?

Reply:
it's only for scala files

Reply:
move to `dev/.scalafmt.conf`?

Reply:
prefer the root folder, to make IDEA pick it up automatically